whalesong/driver_firefox.py

```diff
@@ -177,3 +178,5 @@ def take_screenshot():
                 await self.close()
             except ConnectionRefusedError:
                 pass
+            except WebDriverException:
```

Comment:
Do we need it now?

Reply:
Please reject this pull request. I have tried your fix and it works. I'll make a fresh pull request.